### PR TITLE
Replaced depreciated random_fill w/pseudo_random_fill

### PR DESCRIPTION
**Pull request by hsk81**
The `utility.cpp::random_fill` has apparently been depreciated; removed it, and used `pseudo_random_fill` instead.

### CODE REVIEW

File: src/utility.cpp [hunk 1]
```diff
@@ -90,13 +90,6 @@ string_list numbers_to_strings(const chain::point::indexes& indexes)
     return stringlist;
 }
 
-// Not testable due to lack of random engine injection.
-// DEPRECATED in favor of libbitcoin::pseudo_random_fill.
-void random_fill(data_chunk& chunk)
-{
-    pseudo_random_fill(chunk);
-}
-
```
Comment (evoskuil) [inline src/utility.cpp]: https://github.com/libbitcoin/libbitcoin-explorer/blob/master/include/bitcoin/explorer/utility.hpp#L201-L207

Comment (hsk81) [inline src/utility.cpp]: @evoskuil: `utility.hpp` header has been changed accordingly.

Comment (evoskuil) [inline src/utility.cpp]: Thanks! can you squash commits?

**Comment by evoskuil** [comment]
See https://github.com/libbitcoin/libbitcoin-explorer/pull/472

**Pull request closed without merging**